src/Workspaces/Remote/ServiceHub/Host/AssetProvider.cs

```diff
@@ -52,7 +53,7 @@ internal sealed partial class AssetProvider(Checksum solutionChecksum, SolutionA
         return builder[0];
     }
 
-    public override async ValueTask GetAssetsAsync<T, TArg>(
+    public override async Task GetAssetsAsync<T, TArg>(
```

Comment:
>Task [](http://example.com/codeflow?start=26&length=4)

Out of curiosity, what drove the ValueTask -> Task change in this PR?

Reply:
I needed them as tasks to do the WhenAll work.

Reply:
ahh, gotcha